infrastructure/terraform/modules/gcp-gke/variables.tf

```diff
@@ -0,0 +1,63 @@
+variable "project_id" {
+  type        = string
+  default     = "airy-core"
+  description = "The project defined in gcloud config is airy-core"
+}
+variable "region" {
+  type        = string
+  default     = "us-central1"
+  description = "The region defined in gcloud config is us-central1"
+}
+variable "zone" {
+  type        = string
+  default     = "us-central1-a"
+  description = "The zone defined in gcloud config is us-central1-a"
+}
+variable "gke_num_nodes" {
+  default     = 2
+  description = "number of gke nodes"
+}
+variable "vpc_name" {
+  default = "airy-core-vpc"
+}
+variable "private_subnets" {
+  type        = list(string)
+  default     = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+}
+variable "public_subnets" {
+  type        = list(string)
+  default = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+}
+variable "network" {
+  description = "The VPC network created to host the cluster in"
+  default     = "gke-network"
+}
+variable "subnetwork" {
```

Comment:
This variable is also not used, so not needed I guess. Same for the remaining 5 variables I think?

Reply:
👍🏼